packages/allow-scripts/src/types/common.ts

```diff
@@ -1,15 +1,29 @@
+import type { JsonObject, PackageJson } from 'type-fest';
+
 declare global {
+  export interface PkgLavamoatConfig {
+    ['allowBins']?: JsonObject;
+    ['allowConfig']?: JsonObject;
+    ['allowScripts']?: JsonObject;
+    ['allowedPatterns']?: JsonObject;
+    ['disallowedPatterns']?: JsonObject;
+    ['excessPolicies']?: JsonObject;
+    ['missingPolicies']?: JsonObject;
```

Comment:
```suggestion
    allowBins?: JsonObject;
    allowConfig?: JsonObject;
    allowScripts?: JsonObject;
    allowedPatterns?: JsonObject;
    disallowedPatterns?: JsonObject;
    excessPolicies?: JsonObject;
    missingPolicies?: JsonObject;
```

unsure if there was a reason for writing it that way?

Reply:
also `JSONObject` means "any JSON object", which could be something like  `Record<string, Record<string, number[]|null>>`. Aren't these all just `Record<string, boolean>`?  That's how I see `allowScripts` used, anyhow.

Reply:
> unsure if there was a reason for writing it that way?

Keeping the previous interface intact and unchanged but yeah, I also prefer this form. Updated.

> also JSONObject means "any JSON object", which could be something like Record<string, Record<string, number[]|null>>. Aren't these all just Record<string, boolean>? That's how I see allowScripts used, anyhow.

Yes, but unless I am missing something there is no input validation at runtime beyond that it parses as JSON? Therefore it's not safe to assume values are `boolean` and `JSONObject` is more correct type here, at least until runtime validation is done prior to casting.